pennylane/utils.py

```diff
@@ -226,7 +226,8 @@ def __init__(self, queue, observables, parameters=None):
         """
```

Comment:
Note that the docstring above will have to be updated, since the values no longer contain ints